src/komodo_globals.h

```diff
@@ -42,18 +42,8 @@ int32_t NUM_PRICES; uint32_t *PVALS;
 struct knotaries_entry *Pubkeys;
 
 struct komodo_state KOMODO_STATES[34];
-const uint32_t nStakedDecemberHardforkTimestamp = 1576840000; //December 2019 hardfork 12/20/2019 @ 11:06am (UTC)
-const int32_t nDecemberHardforkHeight = 1670000;   //December 2019 hardfork
 
-const uint32_t nS4Timestamp = 1592146800; //dPoW Season 4 2020 hardfork Sunday, June 14th, 2020 03:00:00 PM UTC
-const int32_t nS4HardforkHeight = 1922000;   //dPoW Season 4 2020 hardfork Sunday, June 14th, 2020 
-
-const uint32_t nS5Timestamp = 1623682800;  //dPoW Season 5 Monday, June 14th, 2021 (03:00:00 PM UTC)
-const int32_t nS5HardforkHeight = 2437300;  //dPoW Season 5 Monday, June 14th, 2021
-
-#define _COINBASE_MATURITY 100
-int COINBASE_MATURITY = _COINBASE_MATURITY;//100;
-unsigned int WITNESS_CACHE_SIZE = _COINBASE_MATURITY+10;
+unsigned int WITNESS_CACHE_SIZE = 100+10; // coinbase maturity plus 10
```

Comment:
a named constant instead of '100'?

Reply:
I would like to replace it with something other than a global. It is only used in wallet functions, so I put it there. See https://github.com/KomodoPlatform/komodo/pull/453/commits/8db798bf3355f791134f7e32e52f00950bb7ee40